### PR TITLE
refactor/snippets: removes need to use new Uint8Array

### PR DESCRIPTION
**Pull request by hunterlester**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @ustulation

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: safe_web_api_playground/static/js/code-snippets/safeImmutableData.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ module.exports = {
       } catch(err) {
         return err;
       }
-      return `Returns ImmutableData data: ${String.fromCharCode.apply(null, new Uint8Array(data))}`;
+      return `Returns ImmutableData data: ${String.fromCharCode.apply(null, data)}`;
```
Comment (bochaco) [inline safe_web_api_playground/static/js/code-snippets/safeImmutableData.js]: This still needs to be converted to `Uint8Array` as it's being returned as `ArrayBuffer`

File: safe_web_api_playground/static/js/code-snippets/safeMutableData.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ module.exports = {
       } catch(err) {
         return err;      
       }
-      return `Name: ${String.fromCharCode.apply(null, new Uint8Array(nameAndTag.name.buffer))}, Tag: ${nameAndTag.type_tag}`;
+      return `Name: ${String.fromCharCode.apply(null, nameAndTag.name.buffer)}, Tag: ${nameAndTag.type_tag}`;
```
Comment (bochaco) [inline safe_web_api_playground/static/js/code-snippets/safeMutableData.js]: We don't need to use `fromCharCode` here

File: safe_web_api_playground/static/js/code-snippets/safeMutableDataEntries.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ module.exports = {
       } catch (err) {
         return err;
       }
-      return `Value: ${String.fromCharCode.apply(null, new Uint8Array(value.buf))}`;
+      return `Value: ${String.fromCharCode.apply(null, value.buf)}`;
```
Comment (bochaco) [inline safe_web_api_playground/static/js/code-snippets/safeMutableDataEntries.js]: You don't need to use `fromCharCode` here

File: safe_web_api_playground/static/js/code-snippets/safeImmutableData.js (hunk 1 above)
Comment (hunterlester) [reply safe_web_api_playground/static/js/code-snippets/safeImmutableData.js]: I'm seeing it returned as Uint8Array:
returns_uint8array

**Comment by hunterlester** [comment]
@bochaco Thank you for telling be about `fromCharCode`. Really nice and clean now that we don't need them anymore. Double-check that you were using the `UpdateRPC` branch of Peruse when you tested this because I'm not receiving `ArrayBuffer` objects returned for any snippet.

File: safe_web_api_playground/static/js/code-snippets/safeCrypto.js [hunk 4]
```diff
@@ -105,7 +105,7 @@ module.exports = {
       } catch (err) {
         return err;
       }
-      return `Nonce generated: ${String.fromCharCode.apply(null, new Uint8Array(nonce.buffer))}`;
+      return `Nonce generated: ${String.fromCharCode.apply(null, nonce.buffer)}`;
```
Comment (bochaco) [inline safe_web_api_playground/static/js/code-snippets/safeCrypto.js]: Do we still need the `fromCharCode` here?

**Pull request merged**